.prettierrc

```diff
@@ -1,11 +0,0 @@
-{
```

Comment:
Why was this removed?

Reply:
I added this file before to recognize .babelrc, .eslintrc, and .prettierrc as json filetypes. But adding the flag `--parser json` is better.